pandas-stubs/core/frame.pyi

```diff
@@ -52,6 +56,7 @@ import xarray as xr
 
 from pandas._libs.missing import NAType
 from pandas._libs.tslibs import BaseOffset
+from pandas._libs.tslibs.period import Period
```

Comment:
This import is unused so you need to remove it.

Strongly suggest setting up a dev environment as described at https://github.com/pandas-dev/pandas-stubs/blob/main/docs/setup.md and testing locally before doing next commit.

Reply:
I did do that, so not sure why pre-commit didn't catch that one. That's a leftover from when I tried to add Series to the overloads as well.

Reply:
Our pre-commit doesn't catch overloads that aren't used.  It's picked up by `pyright` which is not in the pre-commit - just the tests